### DISCUSSION TITLE
Export to json in addition to xml/csv?

### DISCUSSION THREAD

**Discussion by PDXfoster** [Sequel-Ace/Sequel-Ace#894] [Ideas]
Hello! I'd just like to say I'm pleased as punch to find this project - I've been using Sequel Pro for a few years, and really enjoyed the simplicity and lack of trying to do everything. But bugs weren't being fixed, and I was starting to desperately search for other options, and stumbled across Sequel Ace - so thanks for everything you all have done! You've made my life so much easier with this tool. 

I was wondering about the prospect of adding a json export for query results - something like:
![image](https://user-images.githubusercontent.com/45804913/108566955-21b72180-72bc-11eb-9643-fd8bcf6c2da9.png)

I find myself wanting something like this quite often to be able to plug query data into something like Postman/Insomnia and would love it if sequel ace could simplify a part of that workflow.

I'm quite new to contributing to open source, but would love to poke at it if this would be something the project is open to.

Thanks!

**Comment by Jason-Morcos** [comment]
I'm surprised we don't have a bundle that does this! If new don't (I haven't checked), this sounds like it would be a great bundle to write! I also would be happy to review a PR to add this functionality to the core app

**Reply by PDXfoster** [reply]
Awesome! I'm deep in the middle of wrapping my head around an OpenID Connect implementation at the moment, but will take a look and try to scratch something together in the next few weeks.

**Reply by dp-atolls** [reply]
Hi, is there any update on this?
